packages/components/src/components/Label/Label.stories.tsx

```diff
@@ -73,8 +74,26 @@ export const WithDisabledInput: Story = {
 
         return (
             <div className="tw-flex tw-flex-col tw-gap-2">
-                <Label {...args} htmlFor="input" />
-                <STextInput id="input" placeholder="Enter your name" disabled />
+                <Label {...args} htmlFor="input-disabled" />
+                <STextInput id="input-disabled" placeholder="Enter your name" disabled />
+            </div>
+        );
+    },
+};
+
+export const WithSelect: Story = {
+    args: {
+        children: 'Hello World',
+    },
+    render: (args) => {
+        return (
+            <div className="tw-flex tw-flex-col tw-gap-2">
+                {/* The select is using a 'div' as a trigger, this means we have to trigger it manually. */}
+                <Label {...args} htmlFor="select" onClick={() => document.getElementById('select')?.click()} />
```

Comment:
Would be awesome to not have to manually do it (not intuitive) but rather find a way to communicate it to downshift

Reply:
No idea how this would work.

Reply:
It's not intuitive either way. But reversed not so you can leverage an id on the label from the select.